src/org/omegat/core/data/TMXEntry.java

```diff
@@ -28,51 +28,66 @@
 
 package org.omegat.core.data;
 
+import java.util.List;
```

Comment:
This is unused import.

Reply:
Delayed, was true in a patch which will be moved to another pull request